### PR TITLE
Package pollster 2.0

### PR DESCRIPTION
**Pull request by SylvainCorlay**

### CODE REVIEW

File: recipes/pollster/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+test:
+  imports:
+    - pollster
+
+about:
+  home: https://elections.huffingtonpost.com/pollster
+  license: BSD-2-Clause
+  license_family: BSD
+  # License file is not packaged in the pypi tarball
+  # license_file: LICENSE
```
Comment (SylvainCorlay) [inline recipes/pollster/meta.yaml]: Note: opened issue about this. https://github.com/huffpostdata/python-pollster/issues/15

**Comment by SylvainCorlay** [comment]
@beckermr do you know of a recipe that includes the package file?

**Comment by beckermr** [comment]
You mean the license file? A ton of them! Just put a copy in the recipe dir and uncomment the lines above. Or maybe I am not understanding what you mean?

**Comment by SylvainCorlay** [comment]
OK, gotcha thanks.

**Comment by SylvainCorlay** [comment]
@beckermr this is ready to go.

**Pull request merged**